contactaccessrights/CRM/Contactaccessrights/Test/Fabricator/Rights.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Fabricates rights for tests.
+ *
+ * @author Camilo Rodriguez - camilo@compucorp.co.uk
+ */
+abstract class CRM_Contactaccessrights_Test_Fabricator_Rights {

```

Comment:
Well, it's a class that will never be instantiated, and shouldn't be.  That, and most of the fabricators I checked were abstract classes also.  It just made sense to me.